telegram/ext/_contexttypes.py

```diff
@@ -53,7 +53,10 @@ class ContextTypes(Generic[CCT, UD, CD, BD]):
 
     __slots__ = ('_context', '_bot_data', '_chat_data', '_user_data')
 
-    # overload signatures generated with https://git.io/JtJPj
+    """
+     overload signatures generated with
+     https://gist.github.com/Bibo-Joshi/399382cda537fb01bd86b13c3d03a956
+    """
```

Comment:
Please keep this as a comment and not a docstring

Reply:
I have made a new commit, please review. Since I am new to open-source, i didn't know triple quotes meant docstring. Thank You.